src/Chapter1.hs

```diff
@@ -620,8 +633,11 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n =
+    let restAndLast    = divMod (abs n) 10
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `divMod`, cool 😎 

You can see how we could pattern match on the pair 🙂 

```haskell
(aDiv, aMod) = divMod (abs n) 10
```

So you don't need to use `fst` and `snd` 👍🏼